### ISSUE TITLE
Qo

### ISSUE THREAD

**Issue by Psojed** [signus-game/signus#13]

**Comment by nextghost** [comment]
You're right, I should have fixed the default settings and written a README.txt file specifically for the released version. But this would be much easier to fix if there was some serious bug in the game because I need to release version 1.96.1 for this. Two of the source packages are auto-generated by GitHub and leaving them unfixed would be a bad practice, not to mention that Linux distro maintainers would kill me if I replaced the existing source tarballs.

You can switch to fullscreen mode using the standard Alt+Enter keyboard shortcut. The next best thing is maximizing the game window which the game will also remember on startup.

Game language is autodetected from system settings. If your OS locale is Czech or Slovak, the game will run in Czech. In any other locale (or when the Czech language pack isn't installed) the game will run in English. You can force a different language on Linux by setting `LC_MESSAGES` to another locale but I'm not aware of any way to do that on Windows. Hence it's not mentioned in the README.

**Comment by Psojed** [comment]
That means on windows I can force English by simply moving or renaming the cs language folder :)
When I get to actually playing, I'll see if I find something else.

**Comment by nextghost** [comment]
Technically, yes. But why would you want to play a mediocre translation when you clearly understand the real thing?

**Comment by Psojed** [comment]
Yes, I understand the real thing. But I have already experienced the real thing in the original game. I always play my games in English when possible, I've been doing it for a long time now :)

**Comment by nextghost** [comment]
Yes, that makes sense when English is the primary language of the game. But what's the point when English is the translation? You're not missing anything. Trust me, I've translated the whole thing myself. XD

**Comment by RadimKH** [comment]
Hezký den,

nejsem vývojář a už vůbec se v tom nevyznám :D ale ... cca 4 mise před koncem mi to vyhodilo chybu. Tedy zaseklo se to. Spadne a znovu se to načte a zase spadne a načte u posledního tahu. CO s tím? Děkuji

**Comment by nextghost** [comment]
> Hezký den,
> 
> nejsem vývojář a už vůbec se v tom nevyznám :D ale ... cca 4 mise před koncem mi to vyhodilo chybu. Tedy zaseklo se to. Spadne a znovu se to načte a zase spadne a načte u posledního tahu. CO s tím? Děkuji

Založte prosím nový issue a nahrajte do něj soubor `crashguard_saved_state`. Ten najdete na Linuxu v adresáři `~/.signus/` nebo na Windows v adresáři `%APPDATA%\signus\`. Pokud pak chcete pokračovat ve hře ze starší uložené pozice, tak stačí tenhle soubor smazat nebo přejmenovat.
